src/main/java/frc/robot/Constants.java

```diff
@@ -1,6 +1,11 @@
 package frc.robot;
 
+import edu.wpi.first.networktables.NetworkTableInstance;
+
 public final class Constants {
+    //shuffleboard consants:
+    public static final NetworkTableInstance DEFAULT_NETWORK_TABLE_INSTANCE = NetworkTableInstance.getDefault();
```

Comment:
I think this is mixing up the intent of the constants class. I would prefer we explicitly call the NetworkTables.Instance when needed instead of statically initializing it in the constants class